utils/hashdeep/Makefile

```diff
@@ -19,7 +19,7 @@ define Package/hashdeep
   CATEGORY:=Utilities
   TITLE:=Recursively compute hashsums or piecewise hashings
   URL:=http://md5deep.sourceforge.net/
-  MAINTAINER:=Trldp <trldp@outlook.com>
+  MAINTAINER:=Tobe Deprez <trldp@outlook.com>
   DEPENDS:=+libstdcpp
```

Comment:
Does this require libstdcpp or can uclibc++ be used?

Reply:
Tested. Compiles just fine with uclibc++. Please change to $(CXX_DEPENDS) and include $(INCLUDE_DIR)/uclibc++.mk before package.mk

Reply:
If you want to do this, please run-test it first and preferably compare performance before blindly adding it.